weighted_levenshtein/clev.pyx

```diff
@@ -195,8 +212,8 @@ dam_lev = damerau_levenshtein
 
 
 cdef DTYPE_t c_damerau_levenshtein(
-    unsigned char* str1, Py_ssize_t len1,
-    unsigned char* str2, Py_ssize_t len2,
+    int[:] str1, Py_ssize_t len1,
```

Comment:
Does int[:] str1 support str1.size here? Or is it like a C array where you lose the length as soon as you pass it to a function?
If we can still use size then we can remove the len1 parameter.
Same everywhere else.